models/linkages/R/read_restart.LINKAGES.R

```diff
@@ -36,18 +36,23 @@ read_restart.LINKAGES <- function(outdir, runid, stop.time, settings, var.names
   for (i in seq_along(settings$pfts)) {
     pft.names[i] <- settings$pfts[i]$pft$name
   }
-  ens.pft.names <- grep("pft", names(ens))
-  names(ens[[grep("pft", names(ens))]]) <- pft.names
+  #ens.pft.names <- grep("pft", names(ens))
+  #names(ens[[grep("pft", names(ens))]]) <- pft.names
   
   forecast <- list()
 
+  if ("Fcomp" %in% var.names) {
+    forecast[[1]] <- ens$AGB.pft #already has C  #* unit.conv 
```

Comment:
you can softcode list indices e.g. `forecast[[length(forecast)+1]]` might make modifications a little easier next time